gamemodes/terrortown/gamemode/server/sv_roleselection.lua

```diff
@@ -563,6 +563,24 @@ local function SelectForcedRoles(plys, selectableRoles)
 		-- if it's not a selectable role, continue
 		if not roleCount then continue end
 
+		local isBaseRole = roles.GetByIndex(subrole):IsBaseRole()
```

Comment:
`roles.GetByIndex(subrole)` should be stored

Reply:
Generally in the complete roleselection or only in this function, because its only used twice in SelectForcedRoles?